### DISCUSSION TITLE
Very dumb starting question

### DISCUSSION THREAD

**Discussion by tarkhil** [primefaces/community#2996] [PrimeVue]
Hello

I'm starting a new project with PrimeVue + Vite, my main.js is 
```
import './assets/main.css'

import { createApp } from 'vue'
import { createPinia } from 'pinia'
import PrimeVue from 'primevue/config';
import Aura from '@primevue/themes/aura';

import App from './App.vue'
import router from './router'

const app = createApp(App)

app.use(createPinia())
    .use(router)
    .use(PrimeVue, {
        theme: {
            preset: Aura,
            options: {
                cssLayer: {
                    name: 'primevue',
                    order: 'tailwind-base, primevue, tailwind-utilities'
                }
            }
        }
    } )
    .mount('#app')
```

However, I see no styling except buttons. 

`<div class="bg-surface-50 dark:bg-surface-950 px-6 py-20 md:px-12 lg:px-20">` is plainly black on white.

How do I connect proper styling? In examples I see lots of .css/.scss files, but I see no link between themes and those files.

I assume I'm missing something too simple to mention in docs.

**Comment by j0rgedev** [comment]
Hello @tarkhil. As you can see in [sakai vue template](https://github.com/primefaces/sakai-vue/blob/master/src/main.js#L10), you need to import your main css in the main.js file. There is no need to use the cssLayer when setting up primevue
